{"name": "测试用例标题2", "status": "failed", "statusDetails": {"message": "AssertionError: assert 0", "trace": "self = <pytest_test.testcase.test_class.Test_Fixture object at 0x101da9ac8>\n\n    @allure.title(\"测试用例标题2\")\n    @allure.description(\"测试用例2的描述\")\n    @allure.story(\"这里是二级标签:b\")\n    def case_b(self):\n        print(\"----test_b----\")\n        self.step_3()\n>       assert 0\nE       assert 0\n\ntestcase/test_class.py:38: AssertionError"}, "description": "测试用例2的描述", "steps": [{"name": "步骤3", "status": "passed", "start": 1574316744050, "stop": 1574316744050}], "start": 1574316744049, "stop": 1574316744050, "uuid": "6a6295ff-2bcd-4c94-bb73-3a2f2c666edc", "historyId": "50059e70e14d90a6dbcdf0573b43a65e", "fullName": "testcase.test_class.Test_Fixture#case_b", "labels": [{"name": "feature", "value": "这里是一级标签：feature"}, {"name": "story", "value": "这里是二级标签:b"}, {"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_class"}, {"name": "subSuite", "value": "Test_Fixture"}, {"name": "host", "value": "wuyanhongdeMacBook-Pro.local"}, {"name": "thread", "value": "1955-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_class"}]}